tests/integration_tests/test_debug.py

```diff
@@ -78,17 +81,20 @@ def get_val(val):
         DataEntryIds.ORIENTATION_3,
         DataEntryIds.LATITUDE,
         DataEntryIds.LONGITUDE,
-        DataEntryIds.STATE,
     )
     last_values = tuple(map(get_val, vals_to_get))
 
-    assert sensor_inputs[1:] == last_values
+    assert sensor_inputs[1:-1] == last_values
+
+    # Special check for state
+    state_val = get_val(DataEntryIds.STATE)
+    assert STATE_IDS[state_input] == state_val
 
     assert LABLES_UPDATED_EVENT.wait(snapshot) >= 1
 
     assert app.AltitudeLabel.text() == '2.00 m'
     assert app.GPSLabel.text() == '9.00000\xb0, 10.00000\xb0'
-    assert app.StateLabel.text() == '11'
+    assert app.StateLabel.text() == 'DataEntryValues.' + STATE_IDS[state_input].name
```

Comment:
We can get rid of the `DataEntryValues.` if in profiles/label.py we do `.name` when formatting the state label.

Then I think we can mark this as done as well: https://trello.com/c/A23UIJ5j/238-label-type-unit-to-string-format-association

Reply:
Sounds good. Looks like we should be able to mark it as done, yep.